gdx/src/com/badlogic/gdx/graphics/glutils/ETC1.java

```diff
@@ -60,35 +61,75 @@ public ETC1Data (int width, int height, ByteBuffer compressedData, int dataOffse
 			checkNPOT();
 		}
 
+		public ETC1Data (ETC1Data specs) {

```

Comment:
Allows the following move:
```
public ETC1Data (FileHandle pkmFile) {
         		this(ETC1Data.read(pkmFile));
 }
```
Original ```ETC1Data(FileHandle pkmFile)``` creates new instance from file. There are two issues about that:
- file is not closed after reading and locks it's folder in Windows
- I need also reading from input stream;

To address both of this the following changes were made:

1) ```static ETC1Data read(java.io.InputStream inputStream)``` allows reading the the data from stream.
2) ```static ETC1Data read(FileHandle pkmFile)``` 
- properly closes the input file,
- wraps ```read(java.io.InputStream inputStream)``` to avoid code duplication